ex3.html

```diff
@@ -1,555 +1,558 @@
 <!DOCTYPE html>
 <html>
 <head>
+	<link rel="stylesheet" type="text/css" href="ex1_bod_css.css">
+	<link rel="stylesheet" type="text/css" href="ex2_header_css.css">
+	<link rel="stylesheet" type="text/css" href="ex3_bod_css.css">
+	<link rel="stylesheet" type="text/css" href="buttons.css">
 </head>
 <body>
-	<a href="ex2.html"</a>
-		<section>
-			<header> 
-				<img src="logo.jpg" style="float:left; width:200px;height:50px;">
-				<h2><a href="ex2.html" target="_self" style="float: left; margin-left: 20px; margin-right: 20px;">Evaluations</a></h2>
-				<h2><a href="ex3.html" target="_self""> New Evaluation</a></h2>
-				<h4><a href="ex1.html" target="_top" style="float: right;">Log out</a></h4>
-			</header>
-		</section>
-
-		<section>
-			<div id="dm">
-				<form action="/action_page.php">
-					<input class="resize" type="text" placeholder="Candidate" name="Cname">
-					<input class="resize" type="text" placeholder="Interviewer" name="Iname">
-					<input class="resize" type="date" name="ddate">
-
-
-					<style type="text/css">
-						ul {
-							list-style-type: none;
-							
-						}
-						li {
-							float: left;
-							margin-right: 20px;
-						}
-
-						fieldset {
-							margin-bottom: 50px;
-							
-						}
-						textarea {
-							height: 100px;
-							resize: none;
-							width: 100%;
+	<section>
+		<header class="header"> 
+			<img src="logo.jpg" style="float:left; width:200px;height:40px; margin-left: 3%">
+			<h2><a href="ex2.html" target="_self" class="button2" style="margin-left: 7%; margin-right: 0.1%;">Evaluations</a></h2>
```

Comment:
I set it "_self" because i wanted the link to be opened in the same frame as it was clicked, but i find this is default. You can pick _blank(to open in new page), _top(to open in the full body of the window) and _parent(to open it in a frame)